src/main/java/com/minecolonies/coremod/commands/colonycommands/requestsystem/RSCommand.java

```diff
@@ -3,6 +3,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.minecolonies.coremod.commands.AbstractSplitCommand;
 import com.minecolonies.coremod.commands.ColonyCommand;
+import com.minecolonies.coremod.commands.IActionCommand;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.commands.IActionCommand'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)